server/schedule/checker/rule_checker.go

```diff
@@ -70,10 +69,10 @@ func (c *RuleChecker) CheckWithFit(region *core.RegionInfo, fit *placement.Regio
 	c.record.refresh(c.cluster)
 
 	if len(fit.RuleFits) == 0 {
-		checkerCounter.WithLabelValues("rule_checker", "fix-range").Inc()
+		checkerCounter.WithLabelValues("rule_checker", "need-split").Inc()
```

Comment:
Do we need to update the Grafana json also?

Reply:
no need to update. it is just an arbitrary label.